<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoutePropertyView</class>
 <widget class="QWidget" name="RoutePropertyView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>333</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:rgb(30, 33, 39);
	color:rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QLabel" name="TitleLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light Condensed</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;</string>
   </property>
   <property name="text">
    <string>Параметры маршрута</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="SelectedPointGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>195</y>
     <width>320</width>
     <height>160</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-color:rgb(255, 255, 255);
	margin-top: 3px;
	border-top-left-radius: 5px;
    border-top-right-radius: 5px;
    border-bottom-left-radius: 5px;
    border-bottom-right-radius: 5px;
}</string>
   </property>
   <property name="title">
    <string>Выбранная точка маршрута</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="XNameLabel">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>70</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light Condensed</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="YNameLabel">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>110</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light Condensed</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="PointPosNameLabel">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>30</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light Condensed</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Номер</string>
    </property>
   </widget>
   <widget class="QLabel" name="PointPosLabel">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>30</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light Condensed</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="XLineEdit">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>70</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light Condensed</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	selection-background-color:rgb(2, 191,150);
}

QLineEdit:hover
{
    border: 2px solid rgb(2, 191,150);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="YLineEdit">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>110</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light Condensed</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	selection-background-color:rgb(2, 191,150);
}

QLineEdit:hover
{
    border: 2px solid rgb(2, 191,150);
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="PointsNumberNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Всего Точек</string>
   </property>
  </widget>
  <widget class="QLabel" name="PointsNumberLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="RouteLengthNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Длина</string>
   </property>
  </widget>
  <widget class="QLabel" name="RouteTimeNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Время</string>
   </property>
  </widget>
  <widget class="QLabel" name="RouteLengthLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="RouteTimeLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
